### PR TITLE
Upgrade Ember CLI to 3.21

### PR DESCRIPTION
**Pull request by SergeAstapov**

### CODE REVIEW

**Comment by SergeAstapov** [comment]
Tests fail on Ember 3.4, will keep looking into root cause but can't spot right away

**Comment by rwjblue** [comment]
Mind rebasing @SergeAstapov (and possibly updating to latest)?

**Comment by SergeAstapov** [comment]
@rwjblue thanks for the note! Rebased and updated to 3.21 using ember-cli-update

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+* `git clone https://github.com/jmurphyau/ember-truth-helpers.git`
+* `cd ember-truth-helpers`
+* `yarn`
+
+## Linting
+
+* `npm run lint:hbs`
+* `npm run lint:js`
+* `npm run lint:js -- --fix`
+
```
Comment (rwjblue) [inline CONTRIBUTING.md:14]: shoudl be using `yarn` here I think

File: tests/dummy/config/ember-cli-update.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    {
+      "name": "ember-cli",
+      "version": "3.21.2",
+      "blueprints": [
+        {
+          "name": "addon",
+          "outputRepo": "https://github.com/ember-cli/ember-addon-output",
+          "codemodsSource": "ember-addon-codemods-manifest@1",
+          "isBaseBlueprint": true,
+          "options": []
```
Comment (rwjblue) [inline tests/dummy/config/ember-cli-update.json:13]: I think if you add `--yarn` here to these `options` `ember-cli-update` will be able to generate the right content for `CONTRIBUTING.md`.

**Review by rwjblue — COMMENTED** [review]
Only a couple of super minor tweaks left I think, otherwise this is great thank you!

File: tests/dummy/config/ember-cli-update.json (hunk 2 above)
Comment (SergeAstapov) [reply tests/dummy/config/ember-cli-update.json]: @rwjblue didn't know about that, thanks for pointing out!

File: CONTRIBUTING.md (hunk 1 above)
Comment (SergeAstapov) [reply CONTRIBUTING.md]: fixed! Thanks @rwjblue

**Review by rwjblue — APPROVED** [review]

**Pull request merged**